src/workflows/extensions/pages/CreateExtensionPage.js

```diff
@@ -39,6 +48,19 @@ class CreateExtensionPage extends React.PureComponent {
   }
 
   render() {
+    const { userProfile } = this.props;
+
+    if (!userProfile) {
+      return (
+        <div className="content-wrapper">
+          <Alert
```

Comment:
I think maybe we should create a component for this to reuse instead of duping the code all over the place